### PR TITLE
chore(dist): replace G1 config & handle other gc options

### PR DESCRIPTION
**Pull request by MingzhenHan**
Subtask of   https://github.com/apache/incubator-hugegraph/issues/2575

If the user does not specify the GC method, g1 will be used by default.

### CODE REVIEW

File: hugegraph-pd/hg-pd-dist/src/assembly/static/bin/start-hugegraph-pd.sh
Comment (imbajin) [inline hugegraph-pd/hg-pd-dist/src/assembly/static/bin/start-hugegraph-pd.sh:1]: Note: no need to choose g1 for gc (so we could enhance the logic)

File: hugegraph-pd/hg-pd-dist/src/assembly/static/bin/start-hugegraph-pd.sh [hunk 1]
```diff
@@ -101,14 +101,13 @@ fi
...
 # Using G1GC as the default garbage collector (Recommended for large memory machines)
 case "$GC_OPTION" in
-    g1)
+    "")
         echo "Using G1GC as the default garbage collector"
-        JAVA_OPTIONS="${JAVA_OPTIONS} -XX:+UseG1GC -XX:+ParallelRefProcEnabled \
+        JAVA_OPTIONS="${JAVA_OPTIONS} -XX:+ParallelRefProcEnabled \
                       -XX:InitiatingHeapOccupancyPercent=50 -XX:G1RSetUpdatingPauseTimePercent=5"
         ;;
-    "") ;;
```
Comment (imbajin) [inline hugegraph-pd/hg-pd-dist/src/assembly/static/bin/start-hugegraph-pd.sh:109]: seems we lack `zgc` here?

File: hugegraph-store/hg-store-dist/src/assembly/static/bin/start-hugegraph-store.sh [hunk 2]
```diff
@@ -153,14 +153,13 @@ fi
 
 # Using G1GC as the default garbage collector (Recommended for large memory machines)
 case "$GC_OPTION" in
-    g1)
+    "")
         echo "Using G1GC as the default garbage collector"
-        JAVA_OPTIONS="${JAVA_OPTIONS} -XX:+UseG1GC -XX:+ParallelRefProcEnabled \
+        JAVA_OPTIONS="${JAVA_OPTIONS} -XX:+ParallelRefProcEnabled \
                       -XX:InitiatingHeapOccupancyPercent=50 -XX:G1RSetUpdatingPauseTimePercent=5"
```
Comment (imbajin) [inline hugegraph-store/hg-store-dist/src/assembly/static/bin/start-hugegraph-store.sh:159]: same for `zgc`

File: hugegraph-server/hugegraph-dist/src/assembly/static/bin/hugegraph-server.sh [hunk 3]
```diff
@@ -134,9 +134,8 @@ case "$GC_OPTION" in
                                       -XX:ZCollectionInterval=120 -XX:ZAllocationSpikeTolerance=5 \
                                       -XX:+UnlockDiagnosticVMOptions -XX:-ZProactive"
         ;;
-    "") ;;
     *)
-        echo "Unrecognized gc option: '$GC_OPTION', only support 'G1/ZGC' now" >> ${OUTPUT}
```
Comment (imbajin) [inline hugegraph-server/hugegraph-dist/src/assembly/static/bin/hugegraph-server.sh:139]: add info for -> "default use g1" (so as others)

**Review by imbajin — APPROVED** [review]

**Review by VGalaxies — APPROVED** [review]

**Pull request merged**